docs/fpc-lifecycle.txt

```diff
@@ -0,0 +1,60 @@
+@startuml
```

Comment:
can we have a docs/README.md documenting how to generate these very nice uml diagrams? :)

Reply:
will add Makefile and text to README on what to install